halo2-base/src/utils/halo2.rs

```diff
@@ -175,7 +176,7 @@ mod keygen {
             let pk = plonk::keygen_pk2(kzg_params, &circuit, false).unwrap();
             #[cfg(not(feature = "halo2-axiom"))]
             let pk = {
-                let vk = plonk::keygen_vk_custom(kzg_params, &circuit, false).unwrap();
+                let vk = plonk::keygen_vk(kzg_params, &circuit).unwrap();
```

Comment:
Highlighting this change, since I couldnt find `keygen_vk_custom`, set to `keygen_vk` for now.